File: python/SW_Expert_Academy/12368.py
import sys
sys.stdin = open('input.txt', 'r')

test_case_count = int(input())
for t in range(1, test_case_count + 1):
    A, B = map(int, input().split())
    result = A + B
    while (result >= 24):
        result %= 24
    print(f'#{t} {result}')

File: java/SW_Expert_Academy/12368.java
import java.util.Scanner;
import java.io.FileInputStream;

class Solution {
	public static void main(String args[]) throws Exception {
		System.setIn(new FileInputStream("res/input.txt"));

		Scanner sc = new Scanner(System.in);
		int testCaseCount = sc.nextInt();
		for (int i = 1; i <= testCaseCount; i++) {
			byte A = sc.nextByte();
			byte B = sc.nextByte();
			byte result = (A + B) >= 24 ? (byte) ((A + B) % 24) : (byte) (A + B);
			System.out.println("#" + i + " " + result);
		}

	}
}
